routes/productRouter.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const productController = require("../controllers/productController");
+
+const router = express.Router();
+
+router.get("/", productController.getproducts);
```

Comment:
```suggestion
router.get("", productController.getproducts);
```
REST한 API에서는 엔드포인트 끝에 `/`를 사용하는것을 지양하고 있습니다. 수정해주세요~